README.md

```diff
@@ -13,10 +13,14 @@ Made with ❤️ in Cluj-Napoca, Transylvania 🏰
 
 1. Clone the repo    
 1. Run `npm install`
-1. Serve it locally with `npm run serve`
-    * Running _production_ locally `npm run start-prod`
+1. Run
+    * `npm run copy-assets` 
+    * `npm run html-minify`
+    * `npm run bundle`
+    * `npm run build-sw`
+1. It will generate a `public` folder with assets (html, css, js) inside them. Move to `public` folder and run `python -m SimpleHTTPServer 8080`.
```

Comment:
This is no longer needed, you can run `npm run serve` and would spun up a firebase locally served server.